tests/framework/installer/install_helper.go

```diff
@@ -156,12 +162,68 @@ func (h *RookHelper) InstallRook() (err error) {
 	return nil
 }
 
-func NewRookHelper() (*RookHelper, error) {
+func (h *InstallHelper) UninstallRookFromK8s() {
 
-	transportClient := transport.CreateNewk8sTransportClient()
+	k8sHelp := utils.CreatK8sHelper()
+	var err error
+	_, err = k8sHelp.ResourceOperation("delete", path.Join(getPodSpecPath(), rookOperatorFileName))
```

Comment:
is k8sHelp just a wrapper on top of kubectl? If so I think it we be great if it where change to

```
func kubectl(args ...)
```

and it logged and panic inside the function so that this function would look like:

```
kubectl("delete",rookOperatorFileName")
kubectl("delete","rook","cluster","rook")
```